docs/enterprise/metrics-app-overview-page.md

```diff
@@ -30,28 +30,15 @@ The Multi metric column ribbon displays the four values listed below. It shows t
 
 * **Users** - The number of users that performed operations.
 
-## CU over time
+## Capacity utilization and throttling
 
-Displays the CU usage of the selected capacity over time. Use the tabs at the top of the visual to toggle how the visual is displayed:
+It has three features. Use the tabs at the top of the visuals to toggle.
 
-* **Linear** - Display the information using a linear scale that starts at 0 percent.
-
-* **Logarithmic** - Display the information using a logarithmic scale that depends on your CUs consumption.
+### Utilization  
 
-Once you select a column in the chart, you can use the *Explore* button to drill through to the [timepoint](metrics-app-timepoint-page.md) page.
+Displays the CU usage over time. Use the tabs at the top right corner of the visual to toggle how the visual is displayed.
 
-Filters applied to the page in the [Multi metric ribbon chart](#multi-metric-ribbon-chart), affect this chart's display as follows:
-
-* *No filters applied* - Columns display the peak timepoint per hour.
-
-* *Filters are applied* -  The visuals displays every 30 second timepoint.
-
->[!NOTE]
->Peak is calculated as the highest number of seconds from both [*interactive* and *background*](/power-bi/enterprise/service-premium-interactive-background-operations) operations.
-
-To access the [Timepoint](metrics-app-timepoint-page.md) page from this visual, select a timepoint you want to explore and then select **Explore**.
-
-The CU over time chart displays the following elements:
+The utilization chart displays the following elements:
 
 >[!NOTE]
 >Microsoft Fabric is in preview. Some workloads within Fabric are in preview and others are not. Preview workloads are not billable and their consumption is available from the *Background non-billable* and *Interactive non-billable* columns.
```

Comment:
We should not call Fabric as in "Preview" since it will start charging users on 10/1.  It is ok to refer to some workloads as preview / non-billable, though at 10/1 all will be billable (soon after data activator / SQL will be in preview non-billable status.).